crosstool/osx_cc_configure.bzl

```diff
@@ -181,6 +207,15 @@ def configure_osx_toolchain(repository_ctx):
     _compile_cc_file(repository_ctx, wrapped_clang_src_path, "wrapped_clang")
     repository_ctx.symlink("wrapped_clang", "wrapped_clang_pp")
 
+    real_modulemap = None
+    if repository_ctx.os.environ.get("APPLE_SUPPORT_LAYERING_CHECK_BETA") == "1":
+        real_modulemap = "real.modulemap"
```

Comment:
Would having layering_check in the name be more obvious here if this is only used when that's enabled?

Reply:
renamed